packages/@holaplexui-react/src/components/Spinner.tsx

```diff
@@ -7,19 +7,14 @@ interface SpinnerProps {
   className?: string;
 }
 
-const Spinner = ({
-  color,
-  height,
-  width,
-  className,
-}: SpinnerProps): JSX.Element => {
+export const Spinner = ({ color, height, width, className }: SpinnerProps): JSX.Element => {
   return (
     <div className="flex justify-center items-center">
       <div
         className={clsx(
           clsx,
           className,
-          "spinner-border animate-spin inline-block border-1 rounded-full text-gray-600"
+          'spinner-border animate-spin inline-block border-1 rounded-full text-gray-600'
```

Comment:
```suggestion
          'spinner-border animate-spin inline-block rounded-full'
```

border size and color set through spinner-border